nailgun/entities.py

```diff
@@ -5421,6 +5421,7 @@ def __init__(self, server_config=None, **kwargs):
             'organization': entity_fields.OneToManyField(Organization),
             'location': entity_fields.OneToManyField(Location),
             'cacert': entity_fields.StringField(),
+            'default_content_http_proxy': entity_fields.BooleanField(),
```

Comment:
```suggestion
            'default_content': entity_fields.BooleanField(),
```
@Gauravtalreja1, could you provide some functional demonstration with ipython? I tried locally with packit VM for https://github.com/Katello/katello/pull/11183 but I don't get the `default_content_http_proxy` on create nor on read back from API. We may need to add it to the ignore list if it's not returned.

Also, I didn't get the setting updated until I used `default_content` in the create call.